templates/internal/appName/rpc/server.go.tpl

```diff
@@ -13,19 +13,45 @@ package {{ stencil.ApplyTemplate "goPackageSafeName" }} //nolint:revive // Why:
 
 import (
 	"context"
+  {{- $additionalImports := stencil.GetModuleHook "internal/rpc/server.additionalImports" }}
+  {{- if $additionalImports }}
+      {{- range $additionalImports -}}
+  {{ . | indent 2 }}
+      {{- end }}
+  {{- end }}
 )
 
 // Server is the actual server implementation of the API.
 //
 // Note that tracing, logging and metrics are already handled for these
 // methods.
 type Server struct{
+  {{- $additionalHandlerState := stencil.GetModuleHook "internal/rpc/server.additionalHandlerState" }}
```

Comment:
I believe this will conflict with the work @george-e-shaw-iv is doing to enable dependency injection (this also might give you what you want? :D)

Reply:
Yeah I would say hold off here if you can wait until release day (17th) when this is merged - https://github.com/getoutreach/stencil-golang/pull/146

Reply:
ah okay! can do. i'll close this for now and wait for your changes. thanks :D